modules/core/src/test/java/org/apache/ignite/internal/TransactionsMXBeanImplTest.java

```diff
@@ -195,69 +265,181 @@ public void testChangeLongOperationsDumpTimeoutOnImmutableCluster() throws Excep
+        //check that new value after restart
+        checkPropertyValueViaTxMxBean(allNodes, newTimeout, TransactionsMXBean::getLongOperationsDumpTimeout);
 
         newTimeout = 300L;
-        updateLongOperationsDumpTimeoutViaTxMxBean(node0, node -> true, newTimeout);
 
-        Map<IgniteEx, TransactionsMXBean> node2 = singletonMap(startGrid(2), txMXBean(2));
+        updatePropertyViaTxMxBean(allNodes, TransactionsMXBean::setLongOperationsDumpTimeout, newTimeout);
 
-        //check that default value in new node
-        checkLongOperationsDumpTimeoutViaTxMxBean(node0, newTimeout);
-        checkLongOperationsDumpTimeoutViaTxMxBean(node1, newTimeout);
-        checkLongOperationsDumpTimeoutViaTxMxBean(node2, defTimeout);
+        allNodes.putAll(singletonMap(startGrid(2), txMXBean(2)));
+
+        //check that last value in new node
+        checkPropertyValueViaTxMxBean(allNodes, newTimeout, TransactionsMXBean::getLongOperationsDumpTimeout);
     }
 
     /**
-     * Search for the first node by the predicate and change
-     * "Long operations dump timeout." through TransactionsMXBean.
+     * Test to verify the correct change of "Allowance to dump requests from local node to near node,
+     * when long running transaction is found." in an mutable cluster.
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    @WithSystemProperty(key = IGNITE_TX_OWNER_DUMP_REQUESTS_ALLOWED, value = "false")
+    public void testChangeTxOwnerDumpRequestsAllowed() throws Exception {
+        checkPropertyChangingViaTxMxBean(false, true, TransactionsMXBean::getTxOwnerDumpRequestsAllowed,
+            TransactionsMXBean::setTxOwnerDumpRequestsAllowed);
+    }
+
+    /**
+     * Test to verify the correct change of "Threshold timeout for long transactions." in an mutable cluster.
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    @WithSystemProperty(key = IGNITE_LONG_TRANSACTION_TIME_DUMP_THRESHOLD, value = "0")
+    public void testChangeLongTransactionTimeDumpThreshold() throws Exception {
+        checkPropertyChangingViaTxMxBean(0L, 999L, TransactionsMXBean::getLongTransactionTimeDumpThreshold,
+            TransactionsMXBean::setLongTransactionTimeDumpThreshold);
+    }
+
+    /**
+     * Test to verify the correct change of "The coefficient for samples of completed transactions
+     * that will be dumped in log." in an mutable cluster.
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    @WithSystemProperty(key = IGNITE_TRANSACTION_TIME_DUMP_SAMPLES_COEFFICIENT, value = "0.0")
+    public void testChangeTransactionTimeDumpSamplesCoefficient() throws Exception {
+        checkPropertyChangingViaTxMxBean(0.0, 1.0, TransactionsMXBean::getTransactionTimeDumpSamplesCoefficient,
+            TransactionsMXBean::setTransactionTimeDumpSamplesCoefficient);
+    }
+
+    /**
+     * Test to verify the correct change of "The coefficient for samples of completed transactions
```

Comment:
Please use something like {@link #getTransactionTimeDumpSamplesCoefficient} instead of phrases in quotes